pyproject.toml

```diff
@@ -22,12 +23,14 @@ filterwarnings = [
 [tool.cibuildwheel]
 # update skip when numpy wheels become available
 skip = ["*-musllinux_*", "cp310-win32", "cp310-manylinux_i686"]
-test-requires = "pytest"
-test-command = "python -m pytest {package}/tests"
-test-skip = ["*universal2:arm64"]
+test-extras = ["test"]
+test-command = "python {package}/download_data.py && python -m pytest -vv {package}/tests"
+test-skip = ["*_arm64", "*universal2:arm64"]
 # to match numpy, we use manylinux2014 for cp310+
 manylinux-x86_64-image = "manylinux2014"
 manylinux-i686-image = "manylinux2014"
+# Increase the output of pip wheel to -vvv
+build-verbosity = 3
```

Comment:
Is this high verbosity level required? Is build-verbosity = 1 perhaps sufficient? Generating too much output makes it hard to browse the build logs.

Reply:
It was useful level for searching problems on CI from logs. I would leave it.